### ISSUE TITLE
[rush] Rush publish --release-type is ignoring the provided value. 

### ISSUE THREAD

**Issue by lahuey** [Microsoft/web-build-tools#1166]
Hi,

My team is using Rush to build and publish a set of packages that we release to NPM on a quarterly basis. In our most recent release, @johnguy0  noticed that our packages expose APIs that are marked as @internal/@alpha. Could you help us investigate a potential regression with the --release-type argument? 

The commands we use is
`rush publish --publish --pack  --include-all  --version-policy spfxPublic  --release-type publc`
`rush publish --publish --pack  --include-all  --version-policy spfxPublic  --release-type beta`

Thanks!

**Comment by octogonz** [comment]
@lahuey when [this code](https://github.com/Microsoft/web-build-tools/blob/aa890f8fd48942e3ad6f02910dab8a23b05587b8/apps/rush-lib/src/cli/actions/PublishAction.ts#L437) was introduce, I approved it only on condition that it was a very temporary workaround to unblock a high priority SPFx release.  (See [these notes](https://github.com/Microsoft/web-build-tools/pull/620#discussion_r183606219).)  The code is really sinful because it's writing files that don't belong to Rush, without any type validation.

Now seems like a great opportunity for you guys to make good on your promise.  :-P  I'm on Gitter if you need help with the design/implementation of the honorable solution.

**Comment by octogonz** [comment]
Here's a PR to remove the Rush option:  https://github.com/Microsoft/web-build-tools/pull/1198


I'll open a separate PR to simplify the **api-extractor.json** behavior, based on our discussion.

**Comment by octogonz** [comment]
@lahuey the fix was published as API Extractor beta 7.0.37.  You can now specify the exact paths you want for the three files using the **api-extractor.json** settings `untrimmedFilePath`, `betaTrimmedFilePath`, and `publicTrimmedFilePath`.  See [this file](https://github.com/Microsoft/web-build-tools/blob/master/apps/api-extractor/src/schemas/api-extractor-template.json) for details.
